packages/core/util/calculateDynamicBlocks.test.ts

```diff
@@ -62,3 +63,43 @@ test('five', () => {
     }).getBlocks(),
   ).toMatchSnapshot()
 })
+describe('equivalent dynamic and static blocks are at the same offset', () => {
+  const baseView = {
+    width: 980,
+    displayedRegions: [
+      { ...ctgA, start: 0, end: 1000 },
+      { ...ctgA, start: 2000, end: 3000 },
+      { ...ctgA, start: 4000, end: 5000 },
+      { ...ctgA, start: 6000, end: 7000 },
+      { ...ctgA, start: 8000, end: 9000 },
+    ],
+    minimumBlockWidth: 3,
+    interRegionPaddingWidth: 2,
+  }
+  it('when multiple static blocks are offscreen to the left', () => {
+    const view = { ...baseView, offsetPx: 550, bpPerPx: 5 }
+    const dynamicBlocks = calculateVisibleRegions(view).getBlocks()
+    const staticBlocks = calculateStaticBlocks(view).getBlocks()
+    expect(dynamicBlocks.at(1)?.offsetPx).toEqual(staticBlocks.at(6)?.offsetPx)
```

Comment:
These tests failed before the new dynamic blocks calculations, for example the dynamic block's `offsetPx` was 600 and the static block's `offsetPx` was 604, even though they're both for the same region that is fully contained on the screen.